app/containers/Model/CustomModel.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useRef } from 'react';
+import { Modal, View, TouchableWithoutFeedback, Animated, Keyboard } from 'react-native';
+
+import styles from './style';
+
+interface ICustomModalProps {
+	open: boolean;
+	onClose: () => void;
+	children: React.ReactNode;
+}
+
+const CustomModal: React.FC<ICustomModalProps> = ({ open, onClose, children }) => {
+	const opacity = useRef(new Animated.Value(0)).current;
+
+	useEffect(() => {
+		if (open) {
+			Animated.timing(opacity, {
```

Comment:
We should stop using React Native's Animated.
Always use react-native-reanimated.